adam/visualization/positioning.py

```diff
@@ -80,13 +92,33 @@ class AxisAlignedBoundingBox:
     corners at (-1, -1, -1) and (1, 1, 1), giving the box a volume of 2(^3)
     """
 
-    center: torch.Tensor = attrib()  # tensor shape: (3,)
+    center: Parameter = attrib()  # tensor shape: (3,)
     scale: torch.Tensor = attrib()  # tensor shape: (3, 3) - diagonal matrix
     # rotation: torch.Tensor = attrib()
 
     def center_distance_from_point(self, point: torch.Tensor) -> torch.Tensor:
         return torch.dist(self.center, point, 2)
 
+    def distance_from_lowest_corner_to_ground(self) -> torch.Tensor:
+        """
+        Returns distance of z value of minimum (w/r/t Z coordinate) corner of the box from origin's Z value.
```

Comment:
I find this a little hard to read. How about:

"""
Get the position of the lowest corner of the box.

The third coordinate is interpreted as the height relative to the ground at z=0.
"""

Now that I look at this, this method should be called something like `z_coordinate_of_lowest_corner` because it doesn't return the distance to the ground if the lowest corner is below the ground.